services/datamanager/builtin/sync_test.go

```diff
@@ -370,7 +370,8 @@ func TestArbitraryFileUpload(t *testing.T) {
 	for _, tc := range tests {
 		t.Run(tc.name, func(t *testing.T) {
 			// Set up server.
-			tmpDir := t.TempDir()
+			additionalPathsDir := t.TempDir()
+			captureDir := t.TempDir()
```

Comment:
We should be using a temp dir anywhere where we write data in tests. Previously this used /tmp/capture, the value in the config.